### DISCUSSION TITLE
How to build for release?

### DISCUSSION THREAD

**Discussion by mfbx9da4** [wix/react-native-ui-lib#2125] [General]
I'm trying to fork the library so I can test some fixes but I can't see any docs on how to do that and the `"build": "node scripts/build.js",` file is missing.

**Comment by ajp8164** [comment]
Same. I forked this repo made some changes and installed it using npm install. The installation is not running the build scripts to produce the js files in the root directory. What am i doing wrong?

**Reply by ajp8164** [reply]
Haven’t tried yet but maybe we can run the build scripts in the clone dir and then patch node_modules? It’s a hack but might work. I’ll try and post back.

**Comment by ethanshar** [comment]
Try running `npm run build:local`
It should generate a packed version of the lib for you to test.

**Reply by The-White-Fang** [reply]
Can this information be added to `CONTRIBUTING.md`. It will be really appreciated if all the related information can be found there.

**Comment by mfbx9da4** [comment]
I went with using [patch-package](https://github.com/ds300/patch-package) in the end as it means I don't have to think about the build system.

**Reply by ajp8164** [reply]
I like the sentiment of patch-package. I wish it would automagically generate a PR for the source repo though (on its own in a scratch space or wherever). Seems to me that patch-package produces brittle apps. Idk.
